hera_mc/watch_dog.py

```diff
@@ -6,6 +6,40 @@
 """System watch-dogs."""
 
 
+def send_message(subject, msg, to_addr=None, from_addr='hera@lists.berkeley.edu', skip_send=False):
```

Comment:
send_email might be more accurate. Is it ever not an email? Like a SMS or slack?

Reply:
I honestly have no preference, but you are right that that method is for email, so I've renamed.